dev/com.ibm.ws.jdbc_fat_krb5/fat/src/com/ibm/ws/jdbc/fat/db2/JDBCKerberosTest.java

```diff
@@ -76,8 +76,8 @@ public static void setUp() throws Exception {
         server.addEnvVar("DB2_USER", db2.getUsername());
         server.addEnvVar("DB2_PASS", db2.getPassword());
         server.addEnvVar("KRB5_USER", KRB5_USER);
+        server.addEnvVar("KRB5_CONF", krbConfPath.toAbsolutePath().toString());
         List<String> jvmOpts = new ArrayList<>();
-        jvmOpts.add("-Djava.security.krb5.conf=" + krbConfPath.toAbsolutePath());
         jvmOpts.add("-Dsun.security.krb5.debug=true");
```

Comment:
I hope you also test with IBM JDK 8 and the ibm krb5 debug is -Dcom.ibm.security.krb5.krb5Debug

Reply:
I have not tested with IBM JDK 8 yet, but we will test with IBM JDK 8 before we GA the feature